### PR TITLE
feat: Add target attribute to github URL

### PR DESCRIPTION
**Pull request by abritopach**
Hello @scttcper!

I've created this pull request to be able to specify in the component where to open the GitHub URL.

I think it's a very useful option and one that I've found necessary when I've used your component.

I hope you also find it useful and that it can be merge against master.

Thank you very much for your component.

### CODE REVIEW

File: src/lib/gh/gh.component.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ import { getRepo, getUser } from '../util';
   selector: 'gh-button',
   template: `
     <div>
-      <a [href]="buttonHref" class="gh-btn" [attr.aria-label]="text + ' on GitHub'">
+      <a [href]="buttonHref" class="gh-btn" [attr.aria-label]="text + ' on GitHub'"
+      [target]="this.target">
```
Comment (scttcper) [inline src/lib/gh/gh.component.ts:11]:
```suggestion
      [target]="target">
```

File: src/lib/mdo/mdo.component.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { getRepo, getUser } from '../util';
   selector: 'mdo-github-button',
   template: `
     <div class="{{ mainButton }}">
-      <a class="gh-btn" [href]="buttonHref" target="_blank" [attr.aria-label]="text + ' on GitHub'">
+      <a class="gh-btn" [href]="buttonHref" [target]="this.target" [attr.aria-label]="text + ' on GitHub'">
```
Comment (scttcper) [inline src/lib/mdo/mdo.component.ts:10]:
```suggestion
      <a class="gh-btn" [href]="buttonHref" [target]="target" [attr.aria-label]="text + ' on GitHub'">
```

File: src/lib/ntkme/ntkme.component.ts [hunk 3]
```diff
@@ -47,7 +47,7 @@ const svg = {
   template: `
   <div [class.gh-large]="this.size === 'large'">
     <a [href]="buttonHref" class="gh-btn"
-      [attr.aria-label]="text + ' on GitHub'">
+      [attr.aria-label]="text + ' on GitHub'" [target]="this.target">
```
Comment (scttcper) [inline src/lib/ntkme/ntkme.component.ts:50]:
```suggestion
      [attr.aria-label]="text + ' on GitHub'" [target]="target">
```

**Review by scttcper — CHANGES_REQUESTED** [review]

File: src/lib/gh/gh.component.ts (hunk 1 above)
Comment (abritopach) [reply src/lib/gh/gh.component.ts]: Done!

File: src/lib/mdo/mdo.component.ts (hunk 2 above)
Comment (abritopach) [reply src/lib/mdo/mdo.component.ts]: Done!

File: src/lib/ntkme/ntkme.component.ts (hunk 3 above)
Comment (abritopach) [reply src/lib/ntkme/ntkme.component.ts]: Done!

**Pull request merged**